src/cmakeProject.ts

```diff
@@ -1522,6 +1522,7 @@ export class CMakeProject {
         if (isCmakeFile) {
             // CMakeLists.txt change event: its creation or deletion are relevant,
             // so update full/partial feature set view for this folder.
+            this.cmakeListsExists = await fs.exists(path.join(this._sourceDir, "CMakeLists.txt"));
```

Comment:
It doesn't seem like this should be necessary since whenever `this._sourceDir` is set, `this.cmakeListsExists` should also be set.

Reply:
that's right!